packages/identifier-resolution/__tests__/localAgent.test.ts

```diff
@@ -21,7 +21,7 @@ let dbConnection: OrPromise<DataSource>
 let agent: TAgent<IKeyManager & IDIDManager & IIdentifierResolution>
 
 const DID_METHOD = 'did:jwk'
-// const PRIVATE_KEY_HEX = '7dd923e40f4615ac496119f7e793cc2899e99b64b88ca8603db986700089532b'
+119f7e793cc2899e99b64b88ca8603db986700089532b'
```

Comment:
What is happening here?

Reply:
Fixed already.
Bram requested to remove these, but his message rendered over my cursor in IJ must have messed up my selection